app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -29,6 +29,11 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private var mLocationPermissionGranted = false
     private val LOCATION_PERMISSION_REQUEST_CODE = 1234
     private lateinit var mFusedLocationProviderClient: FusedLocationProviderClient
+
+    class FusedLocationProviderClient {
```

Comment:
My apologies, I thought I took out this class before I committed. If you were to remove this class this app builds and accesses the map features.

Reply:
I'd remove and recommit... or revert the commit.